src/en/kunmangato/src/eu/kanade/tachiyomi/extension/en/kunmangato/KunMangaToChapterResourcesDto.kt

```diff
@@ -0,0 +1,21 @@
+package eu.kanade.tachiyomi.extension.en.kunmangato
+
+import kotlinx.serialization.Serializable
+
+@Serializable
+data class KunMangaToChapterResourcesDto(
```

Comment:
Remove if possible, `data class` creates additional methods

```suggestion
class KunMangaToChapterResourcesDto(
```

Reply:
Fixed